### DISCUSSION TITLE
Notifications Library is amazing.

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#14] [Feedback]
Just some feedback on the notifications library. It is fantastic!
Having the queue is great. The alerts are very customisable - exactly what I need.

Are there any plans to allow for more advanced interaction with notifications?

For example have a 'confirm' notification with yes/no buttons - use case could be if a user clicks `logout` - have a notification pop up with confirmation.
Another use-case could be clicking to edit text and a notification pops up with a text entry and submit button to allow text input ala `prompt()`.

**Comment by rtivital** [comment]
Thanks for bringing that up, just tested and [released this feature](https://mantine.dev/others/notifications/#react-node-in-notification-message) in [1.0.2](https://github.com/mantinedev/mantine/releases/tag/1.0.2)

**Reply by cstrat** [reply]
Brilliant thanks! This is great
